Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            
+        }
+        public static int PromptInt(string message)
+        {
+            int Newmessage;
```

Comment:
You can initialize and declare at the same time. Single line:
`int newMessage = int.TryParse( ...`

Reply:
And use camelCase for variable names. So it should be newMessage, not Newmessage